python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -475,7 +476,9 @@ def predict(self,
                                               shard_size=batch_size)
 
             pred_shards = self._predict_spark_xshards(xshards, init_params, params)
-            result = convert_predict_xshards_to_dataframe(data, pred_shards)
+            result = convert_predict_xshards_to_dataframe(data,
```

Comment:
what about SparkXShards? we also need to support that

Reply:
We have supported this in previous pr https://github.com/intel-analytics/BigDL/pull/7704/files#diff-ce59ed5b2127ca719be83567a5d7443351f2491296219720aade783069860748R155-R179

Reply:
Our xshards are still working the same as before, and we will plan to support this in the future.